k9mail/src/test/java/com/fsck/k9/view/MessageHeaderTest.java

```diff
@@ -0,0 +1,56 @@
+package com.fsck.k9.view;
+
+import android.util.AttributeSet;
+
+import com.fsck.k9.K9RobolectricTestRunner;
+import com.fsck.k9.mail.Address;
+import com.fsck.k9.mail.Message;
+import com.fsck.k9.mail.internet.MimeMessage;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RuntimeEnvironment;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.mockito.Mockito.mock;
+
+@RunWith(K9RobolectricTestRunner.class)
+public class MessageHeaderTest {
+
+    private static final Address FROM_ADDRESS = Address.parse("from@example1.com")[0];
+    private static final Address SENDER_ADDRESS = Address.parse("sender@example2.com")[0];
+
+    private MessageHeader messageHeader;
+
+    @Before
+    public void setUp() throws Exception {
+        messageHeader = new MessageHeader(RuntimeEnvironment.application, mock(AttributeSet.class));
+    }
+
+    @Test
+    public void withSender_shouldShowSender_shouldReturnTrue() {
```

Comment:
You got the order wrong. It should be `shouldShowSender_withSender_shouldReturnTrue`. Same with the other test method names.